rpcserverhandlers_test.go

```diff
@@ -3112,6 +3113,22 @@ func TestHandleGetCurrentNet(t *testing.T) {
 	}})
 }
 
+func TestHandleGetDifficulty(t *testing.T) {
+	t.Parallel()
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleGetDifficulty: ok",
+		handler: handleGetDifficulty,
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.bestSnapshot.Bits = chaincfg.MainNetParams().PowLimitBits
```

Comment:
Minor, but you can use `defaultChainParams` instead of `chaincfg.MainNetParams()` here